src/config/theme.ts

```diff
@@ -28,10 +28,12 @@ declare module '@mui/material/styles' {
   interface Palette {
     borders: PaletteColor;
     alerts: Record<string, string>;
+    linkColor: string;
   }
   interface PaletteOptions {
     borders: SimplePaletteColorOptions;
     alerts: { lightWarningBackground?: string };
+    linkColor: string;
```

Comment:
I think `Color` is redundant - since this is a `Palette`, everything in it should be a color, right? Can we just call it `links`?